storm-core/src/jvm/backtype/storm/messaging/netty/Client.java

```diff
@@ -134,7 +128,7 @@
         LOG.info("creating Netty Client, connecting to {}:{}, bufferSize: {}", host, port, bufferSize);
         int messageBatchSize = Utils.getInt(stormConf.get(Config.STORM_NETTY_MESSAGE_BATCH_SIZE), 262144);
 
-        maxReconnectionAttempts = Utils.getInt(stormConf.get(Config.STORM_MESSAGING_NETTY_MAX_RETRIES));
+        int maxReconnectionAttempts = Utils.getInt(stormConf.get(Config.STORM_MESSAGING_NETTY_MAX_RETRIES));
```

Comment:
> It would be nice to deprecate this and file a follow on JIRA so that we can use a better named/documented config in the backoff calculation

Disregard: I see from the above comment what we intend to do.